src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionFeedbackPathSettings.tag

```diff
@@ -9,7 +9,7 @@
 
 <div class="col-sm-12 padding-15px margin-bottom-15px background-color-light-green">
     <div class="col-sm-12 padding-0 margin-bottom-7px">
-        <b>Feedback Path</b> (Who is giving feedback about whom?)
+        <b class="feedbackPathTitle">Feedback Path</b> (Who is giving feedback about whom?)
```

Comment:
Class names should be lowercase and delimited by hyphens based on our naming conventions, do edit this :)